api/nakadi-event-bus-api.yaml

```diff
@@ -0,0 +1,1145 @@
+        provides information about the violation it will be included in the Problem response.
+        If the `EventType` defines schema validation it will be performed at this moment.
+        **TBD behaviour in case of violation within a stream of events**)
+
+        1. Once the validation succeeded, the content of the Event is updated according to the enrichment
+        rules in the order the rules are defined in the `EventType`.
+        No preexisting value might be changed (even if added by an enrichment rule). Violations on this will
+        force the immediate **rejection** of the Event with a 409 Conflict status. The invalid overwrite
+        attempt will be included in the respose's Problem object.
+
+        1. The incoming Event's relative ordering is evaluated according to the rule on the
+        `EventType`. Failure to evaluate the rule will **reject** the Event.
+        **Restriction**: POSTing Events based on its `EventType` requires it to have a
+        `PartitioningStrategy` defined. Violating this condition will produce a 405 Not allowed.
+
+      parameters:
+        - name: name
+          in: path
+          type: string
+          description: Name of the EventType
+          required: true
+      responses:
+        '200':
+          description: Ok
+        '400':
+          description: Bad request
+          schema:
+            $ref: '#/definitions/Problem'
+        '401':
+          description: Client is not authenticated
+          schema:
+            $ref: '#/definitions/Problem'
+        '404':
+          description: EventType not found.
+          schema:
+              $ref: '#/definitions/Problem'
+        '405':
+          description: Not allowed.
+          schema:
+              $ref: '#/definitions/Problem'
+        '409':
+          description: Conflict.
+          schema:
+              $ref: '#/definitions/Problem'
+        '422':
+          description: Unprocessable Entity
+          schema:
+            $ref: '#/definitions/Problem'
+        '500':
+          description: Server error
+          schema:
+            $ref: '#/definitions/Problem'
+
+  /topics:
+    get:
+      description: Lists all known `Topics` of this Nakadi cluster
+      tags:
+        - monitoring
+        - management-api
+      responses:
+        '200':
+          description: The request was successful. Returns a list `Topic`s in the response body.
+          schema:
+            type: array
+            description: An array of topics
+            items:
+              $ref: '#/definitions/Topic'
+        '401':
+          description: Client is not authenticated
+          schema:
+            $ref: '#/definitions/Problem'
+        '500':
+          description: Server error
+          schema:
+            $ref: '#/definitions/Problem'
+
+        '503':
+          description: Not available
+          schema:
+            $ref: '#/definitions/Problem'
+
+  '/topics/{topic}/events':
+    get:
```

Comment:
Since it's stressed that the Topic API is "low-level", is it also planned to have a "high-level" way of fetching events like a `GET /event-types/{name}/events` or does it make no sense?